### ISSUE TITLE
Run as admin still required in 2.4 despite docs?

### ISSUE THREAD

**Issue by RoxKilly** [OpenVPN/openvpn-gui#129]
The docs mention that as of OpenVPN 2.4 (GUI 11.4.0.0), the GUI no longer needs to be run as administrator so long as the `OpenVPN Interactive Service` is running.

However, I am unable to test this because whenever I run `C:\Program Files\OpenVPN\bin\openvpn-gui.exe`, it asks permission to run as admin as it did for previous versions.

I've tried to open the EXE file properties and change this requirement, but the box *"Run as admin"* is checked and read-only, even when I edit properties from an Admin account (see the last checkbox in the attached image)

![6](https://cloud.githubusercontent.com/assets/12346501/22679149/eaeb57c0-eccd-11e6-9630-9d5d4fb15525.png)
.
I'm using Windows 7 x64, and I upgraded in-place from 2.3.12 (GUI 10)

**Comment by mattock** [comment]
@RoxKilly : "Run as admin" is _not_ required. That said, I have no clue why "Run this program as an administrator" is grayed out for you. If you're an admin, you should be able to reset the flag.

Very recently we had a discussion regarding the "Run as admin" flag for OpenVPN GUI. The thing is that one can set the flag from two places:

* The shortcut
  * "Run as admin" flag stored in the shortcut file itself
* The executable
  * "Run as admin" flag stored in the registry

The _shortcut_ flag probably gets erased when OpenVPN is upgraded, but the _executable_ flag does not. I could not find the original discussion - @selvanair : any idea where it is?

**Comment by selvanair** [comment]
@RoxKilly: To add to what mattock wrote, the  installer does not set the run as admin flag on the executable or shortcut. However, if you had a previous installation where you had set the executable to run as admin, it could survive after a new install. For a discussion of this issue see https://community.openvpn.net/openvpn/ticket/811

To remove the flag you have two options:
(i) As administrator, right click on the executable, got to properties and click the button "change settings for all users" as seen the screenshot you posted. Then you will get a dialog where you can remove the run-as-admin flag.
(ii) Uninstall and reinstall openvpn

Going forward do not set either the shortcut or exec to run as admin.

**Comment by RoxKilly** [comment]
@selvanair  @mattock thanks for your help. It is useful to know that the flag was not set by OpenVPN.  The admin flag was set on the exe, not the shortcut. Perhaps I had set it when I first started using OpenVPN a couple of years back I don't remember.  

Using *"Change settings for all users"* to uncheck "run as admin" made no difference. The program still prompted for admin priviledges, and in exe properties >> compatibility, the "*Run as admin*" checkbox was still checked and greyed out (so cannot be unchecked).

To get things to work, I needed to:
1. change my user account from standard user to admin user
2. restart the PC (this is the step I had missed)
3. Now the checkbox is no longer greyed out, so I could uncheck it.

Remember to change the user account back to standard user.

Feel free to close the issue.
